tests/e2e/tests/e2e_install_che_operatorhub/InstallCheByOperatorHub.spec.ts

```diff
@@ -65,6 +64,10 @@ suite('E2E', async () => {
         });
     });
 
+    if (TestConstants.TS_SELENIUM_UPDATE_INSTALLED_APPLICATION) {
```

Comment:
I haven't had a time to try this, but do I understand it correctly, that this section makes it that no further suites/tests are executed, when `TS_SELENIUM_UPDATE_INSTALLED_APPLICATION=true`?
If yes, this doesn't seem right to me, as there is no validation, that the update was succesfull.
Maybe even more I think updating should have separate spec.ts file to more distinguish these two scenarios (install vs update). And this variable wouldn't be needed at all.

Reply:
@dmytro-ndp I don't mind..

@rhopp Really this change needs to check and automate a private case updating CHE6 to CHE7 using operatorHub which is not supported automate update. So we should do defined actions on web-console OCP. Actually we have a flow: install an application using old operator,    remove subscription old operator and then installed new operator to an existing application. 
But I'm not sure that we need a separate E2E test for this, because it will fully duplicate the part of steps from already exist 'InstallCheByOperatorHub.spec.ts'. 
If the `TS_SELENIUM_UPDATE_INSTALLED_APPLICATION=true` the `InstallCheByOperatorHub.spec.ts` performs only part steps needed to subscribe to a new operator and ends.
You're right about verifying success updating, but I think it should be checked using `oc` client in the deployment or something like that in the job which tests this scenario.

Reply:
@rhopp 
Done.